src/DrTests/DrTestsPlugin.class.st

```diff
@@ -66,14 +81,12 @@ DrTestsPlugin >> resultButtonLabel [
 
 { #category : #api }
 DrTestsPlugin >> run [
```

Comment:
In fact, `#run` method can be totally removed it has been replaced by `#runForConfiguration:`.